packages/components/src/mobile/alt-text-settings/styles.scss

```diff
@@ -0,0 +1,32 @@
+.altTextEditor {
+	padding: 14px 0;
+	height: 80px;
+}
+
+.altTextEditorDark {
+	color: $white;
+}
+
+.altTextFootnote {
+	font-size: 12px;
```

Comment:
Is there a sass variable that we can use for this?

Reply:
This was changed to 12px following [this discussion](https://github.com/WordPress/gutenberg/pull/29670/files/ff711fa39da42c484e5f0725f284851a28ab14b8#r591931227) and I wasn't able to find a variable for 12px under [the _variables.scss file](https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_variables.scss). I was a bit wary of creating a new variable as I thought that might be part of a larger conversation, as I think @iamthomasbishop alluded to in one of his replies ("worry about the neutral type-scale at a later time"). I'd be interested to hear any further thoughts from you both on what to do in this scenario. Does this perhaps warrant a separate GitHub issue to ensure this is addressed at a later time?